testing/python_lib/test_failscale.py

```diff
@@ -34,14 +34,21 @@ def __init__(self, *args, **kwargs):
             'skip-conn-check': False
         })
 
+    def tearDown(self):
```

Comment:
what's this for?  (i.e., why not let the superclass handle it?)

Reply:
Outdated code